src/Core/Helpers/WebServiceHelper.cs

```diff
@@ -40,15 +40,14 @@ public static async Task<T> SafeServiceCallAsync<T>(Func<Task<T>> call, ILogger
                 var innerException = e.InnerException as System.Net.WebException;
                 logger.WriteLine(CoreStrings.SonarQubeRequestFailed, e.Message, innerException?.Message);
             }
-            catch (TaskCanceledException)
```

Comment:
Miscellaneous: `TaskCanceledException` is a subclass of `OperationCanceledException` so there's no need to specify it separately - it will be caught by the `catch(OperationCanceledException)`.